docs/messaging/messaging-spans.md

```diff
@@ -359,92 +397,85 @@ All attributes that are specific for a messaging system SHOULD be populated in `
 
 ### Topic with multiple consumers
 
-Given is a process P, that publishes a message to a topic T on messaging system MS, and two processes CA and CB, which both receive the message and process it.
-
-```
-Process P:  | Span Prod1 |
---
-Process CA:              | Span CA1 |
---
-Process CB:                 | Span CB1 |
+Given is a process P, that publishes a message to a topic T on messaging system MS, and two processes CA and CB, which both receive the message.
+
+```mermaid
+flowchart LR;
+  subgraph PRODUCER
+  direction TB
+  P[Span Publish A]
+  end
+  subgraph CONSUMER1
+  direction TB
+  R1[Span Receive A 1]
+  end
+  subgraph CONSUMER2
+  direction TB
+  R2[Span Receive A 2]
+  end
+  P-. link .-R1;
+  P-. link .-R2;
+
+  classDef normal fill:green
+  class P,R1,R2 normal
+  linkStyle 0,1 color:green,stroke:green
 ```
 
-| Field or Attribute | Span Prod1 | Span CA1 | Span CB1 |
+| Field or Attribute | Span Publish A | Span Receive A 1| Span Receive A 2 |
 |-|-|-|-|
-| Span name | `"T publish"` | `"T process"` | `"T process"` |
-| Parent |  | Span Prod1 | Span Prod1 |
-| Links |  |  |  |
+| Span name | `T publish` | `T receive` | `T receive` |
+| Parent | | | |
+| Links |  | `T publish` | `T publish` |
 | SpanKind | `PRODUCER` | `CONSUMER` | `CONSUMER` |
 | Status | `Ok` | `Ok` | `Ok` |
 | `server.address` | `"ms"` | `"ms"` | `"ms"` |
 | `server.port` | `1234` | `1234` | `1234` |
 | `messaging.system` | `"rabbitmq"` | `"rabbitmq"` | `"rabbitmq"` |
 | `messaging.destination.name` | `"T"` | `"T"` | `"T"` |
-| `messaging.operation` |  | `"process"` | `"process"` |
-| `messaging.message.id` | `"a1"` | `"a1"`| `"a1"` |
-
-### Batch receiving
-
-Given is a process P, that publishes two messages to a queue Q on messaging system MS, and a process C, which receives both of them in one batch (Span Recv1) and processes each message separately (Spans Proc1 and Proc2).
-
-Since a span can only have one parent and the propagated trace and span IDs are not known when the receiving span is started, the receiving span will have no parent and the processing spans are correlated with the producing spans using links.
-
+| `messaging.operation` | `"publish"` | `"receive"` | `"receive"` |
+| `messaging.message.id` | `"a"` | `"a"`| `"a"` |
+
+### Batch delivering
+
+Given is a process P, that publishes two messages to a queue Q on messaging system MS, and a process C, which gets both of them delivered in one batch (Span Recv1) and processes each message separately.
+
+Since a span can only have one parent, the `deliver` span will have no parent will be correlated with the producing spans using links.
```

Comment:
```suggestion
Since a span can only have one parent, the `deliver` span will have no parent and will be correlated with the producing spans using links.
```